valve-monitor/firmware/src/main.cpp

```diff
@@ -29,14 +29,9 @@
 
 #define LEAK_THRESHOLD 6
 
-// Define USE_VALVE to compile in valve control support. If not defined, it's
-// turned on by default.
-#ifndef USE_VALVE
-#define USE_VALVE 1
-#endif
-
 // Replace with your product UID.
 // #define PRODUCT_UID "com.my-company.my-name:my-project"
+#define PRODUCT_UID "com.blues.hroche:quickstart"
```

Comment:
Can we leave this undefined by default, like this: https://github.com/blues/app-accelerators/blob/main/08-power-monitoring-and-control/firmware/src/app.h#L12

Reply:
Also please note that we have a short link, just in case the docs layout changes.